packages/material-ui/src/index.d.ts

```diff
@@ -74,6 +74,9 @@ export {
   WithTheme,
 } from './styles';
 
+export * from './styles/colorManipulator';
```

Comment:
Ah, great idea! What do you think of exporting everything that `./styles/index.js` export in the root index (instead of cherry-picking)?

Reply:
We should probably do that with all the exports, i'll update the PR later